ClientPlugin/Settings/Layouts/None.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Sandbox.Graphics.GUI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using VRageMath;
+
+namespace ClientPlugin.Settings.Layouts
+{
+    internal class None : Layout
```

Comment:
What is the reason to use the `None` layout first, then setting the `Simple` one right before opening the dialog?

Reply:
None is the logical default layout (no layout).
The plan is to add more layouts in the future and the user selects the one they want.
They can also select one after the object is created to change layouts on the fly.